src/constants.ts

```diff
@@ -19,6 +20,21 @@ export const DATA_EXTENSIONS = [".mkv", ".avi", ".mp4", ".ts", ".flac", ".mp3"];
 
 export const TORRENT_CACHE_FOLDER = "torrent_cache";
 
+export const ZOD_ERROR_MAP: z.ZodErrorMap = (error, ctx) => {
+	switch (error.code) {
+		case z.ZodIssueCode.invalid_union:
+			return {
+				message: error.unionErrors
+					.reduce((acc, error) => {
+						error.errors.forEach((x) => acc.push(x.message));
+						return acc;
+					}, [])
+					.join("; "),
+			};
+	}
+
+	return { message: ctx.defaultError };
+};
```

Comment:
could this live in `zod.ts`?

Reply:
It can, I just figured since it wasn't used in zod.ts, but in cmd.ts that putting it with the other constants would be appropriate. 

I'll move it and update the imports.